api/namex/services/queue.py

```diff
@@ -107,8 +102,15 @@ async def connect(self):
             if not ctx.nats.is_connected:
                 self.stan_options = {**self.stan_options, **{'nats': ctx.nats}}
                 await ctx.nats.connect(**self.nats_options)
+                # Update the client_id if nats is reconnecting
+                self.stan_options['client_id'] = self.get_client_id()
```

Comment:
Why is this being changed?

Reply:
In namex-api it doesn't reconnect frequently but in namex-pay it reconnects often. In that part of the code it checks if nats is not connected and uses the same stan's client_id. When it happens it raises an error saying that the client_id is already connected.

Reply:
you've seen that in the logs?
I haven't seen it reconnecting unless the network has an error.
Over the past 14 days it has had 1 reconnect in Prod